identity-android/src/main/java/com/android/identity/android/mdoc/deviceretrieval/DeviceRetrievalHelper.java

```diff
@@ -713,7 +715,7 @@ public Builder(@NonNull Context context,
          */
         public @NonNull Builder useReverseEngagement(@NonNull DataTransport transport,
                                                      @Nullable byte[] readerEngagement,
-                                                     List<OriginInfo> originInfos) {
+                                                     @Nullable List<OriginInfo> originInfos) {
```

Comment:
I would prefer not having this Nullable as especially in Kotlin it's preferable to avoid nullable types... instead we should just allow it being the empty list. This way you also won't have to make nullable types in the app for passing this (can just pass an empty list instead).